source/mainnet/docs/smart-contracts/best-practices/index.rst

```diff
@@ -0,0 +1,17 @@
+.. _best-practices:
+
+==============
+Best Practices
+==============
+
+Concordium, with its unique approach to identity verification and privacy, provides a robust platform for secure and scalable smart contract development.
+However, creating efficient, reliable, and cost-effective smart contracts requires adherence to best practices that maximize performance while minimizing risks and costs.
+This following artilces in this section are dedicated to guiding developers through the best practices for building high-quality smart contracts on Concordium.
```

Comment:
These following articles
delete last bit 'in Concordium'  - its clear enough

Reply:
Fixed.